packages/teleport-plugin-react-app-routing/src/index.ts

```diff
@@ -31,8 +37,10 @@ export const createPlugin: ComponentPluginFactory<AppRoutingComponentConfig> = (
 
   const reactAppRoutingComponentPlugin: ComponentPlugin = async (structure) => {
     const { uidl, dependencies, options } = structure
+    // @ts-ignore-next-line
```

Comment:
I would drop this in favor of explicit initialization for all the 3 values you are destructuring below:

```const { useFolderStructure, disableDOMInjection, flavour } = options.meta || { useFolderStructure: false, disableDOMInjection: false, ... }```